app/src/main/res/layout/include_backable.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+
+<android.support.v7.widget.Toolbar xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/toolbar"
+    android:layout_width="match_parent"
+    android:layout_height="?attr/actionBarSize"
+    android:background="@color/colorPrimary"
+    android:theme="@style/ThemeOverlay.AppCompat.ActionBar"
+    tools:showIn="@layout/fragment_setting" />
```

Comment:
Is this reference to `@layout/fragment_setting` intentional?  Have any side-effects been observed?

Reply:
Reading the [reference](https://developer.android.com/studio/write/tool-attributes#toolsshowin), it seems this only affects the Android Studio layout editor, to help with visually editing the toolbar. It's probably fine, but might be worth noting this.

Reply:
👍 

This is used in multiple places, so I've deleted the `showIn` attribute.